src/main/resources/application.yml

```diff
@@ -2,34 +2,34 @@ server:
   port: 9000
 ---
 spring:
-  profiles: LOCAL, default, test, MACOS
+  profiles: LOCAL, default, test
   authKey: mockAuthKey
   datasource:
-    url: jdbc:postgresql://localhost:5432/cag_db?currentSchema=cag_backend_api
-    username: cag_admin
-    password: ${PSQL_PASSWORD:cag_pass}
+    url: jdbc:postgresql://ec2-54-159-113-254.compute-1.amazonaws.com:5432/d1s2s8407s5lms?currentSchema=public
+    username: axdarkeikdwxvm
+    password: b9d7b014718e89ea3d2cad53522f73c25e8c511f6bdc006c9ddc5baf1f8aee49
```

Comment:
Fun fact: Never do this. The database user and password information should be kept secret, and even the host address is also typically something you don't want to reveal. In this case it's a free database that isn't tied to any live data. If a hacker wants to spend there day connecting to our database and deleting everything - they can go right ahead. We can delete and create new users in Heroku in less than 5 mins and also when you run our app we have a flyway migration manager that builds our database from scratch everytime we launch our app. What I'm saying is - we're safe, don't worry! The PROD AWX user and password we will definitely be more careful with!

Reply:
Why is the host address considered important information?

Reply:
The host address will reveal to everyone in the world where our database is running - in this case on AWS EC2. So if you're a hacker, you can go and search the internet for vulnerabilities of EC2 databases and maybe find a way to break some of our stuff. If we hid the host address - a hacker wouldn't know where to look or what platform is running ou database. In their eyes, we could be on Azure, AWS, Oracle, or a hundred other places. 

You can run this is you do "git pull" on your local machine, resolve any conflicts if you have them, and then switch branches using "git checkout DEV-135-Heroku-db-setup" to switch to this branch. I'll merge the branch now so you can just run the project from the "main" branch so you don't need to switch to this one!